src/main/play-doc/developer/BundleConfiguration.md

```diff
@@ -136,6 +136,80 @@ The declaration of interest is the `jms-status` component. ConductR provides a `
 
 Docker becomes relevant when there are specific runtime dependencies that are different to ConductR's host OS environment. In particular if a binary program that does not use the JVM is required to be launched from a bundle then it becomes more likely to benefit from using a Docker container.
 
+#### When you cannot Dockerize existing services
+
+If you cannot run a Linux service in Docker but would still like the services to be managed by ConductR, it may be possible to run services without a container. To do this, create an application bundle of the full service installation folder and a configuration bundle containing any scripting needed to control the service. 
+
+For example, to a run a single instance of [Redis](http://redis.io/) under ConductR without a container, start with a built version of redis. For this example `redis/redis-stable` will be the top level folder of the Redis installation containing the Redis README, MANIFESTO and COPYING. Redis will be executing natively on the member node and must be built for the target node's distribution. We will use roles to ensure that this bundle only runs on specific nodes.
+
+In the `redis` folder, add the following as a file named `bundle.conf`.
+
+```bash
+version               = "3.0.4"
```

Comment:
The bundle's version should be "1.1.0".

What you've got here is a 1.0.0 bundle format. For 1.1.0:

```
version              = "1.1.0"
name                 = "redis-broker"
compatibilityVersion = "3"
system               = "redis3"
systemVersion        = "3"
```